docs/glossary.md

```diff
@@ -23,17 +23,17 @@ To help you navigate the platform more effectively, familiarize yourself with th
  |                                                                                                                            |**Build Pipeline** - a Tekton pipeline that builds a corresponding codebase branch in the Codebase.|
  |                                                                                                                            |**Build Stage** - a stage that takes place after the code has been submitted/merged to the repository of the main branch (_the pull request from the feature branch is merged to the main one, the Patch set is submitted in GitHub/GitLab_).|
  |                                                                                                                            |**Code Review Pipeline** - a Tekton pipeline that inspects the code candidate in the Code Review tool, triggered on Pull Request created/updated event.|
  |                                                                                                                            |**Code Review Stage** - a stage where code is reviewed before it goes to the main branch repository of the version control system (_the commit to the feature branch is pushed, the Patch set is created in GitHub/GitLab_).|
  |                                                                                                                            |**Deploy Pipeline** - a Tekton pipeline that is responsible for the CD Pipeline Stage deployment with the full set of applications and autotests.|
- |                                                                                                                            |**Deployment Stage** - a part of the Continuous Delivery where artifacts are being deployed to environments.|
+ |                                                                                                                            |**Environment** - a part of the Continuous Delivery where artifacts are being deployed to.|
  |                                                                                                                            |**CI Pipelines** - an orchestrator for stages that is responsible for the common technical events, e.g. initialization, in Tekton pipeline.|
- |                                                                                                                            |**Environment** - a part of the stage where the built and packed into an image application are deployed for further testing. It`s possible to deploy several applications to several environments (Team and Integration environments) within one stage.|
+ |                                                                                                                            |**Environment** - a namespace where the built and packed into an image applications are deployed for further testing. It is possible to deploy several applications to several environments (Team and Integration environments) within one deployment flow.|
```

Comment:
a part of the Deployment Flow

Please use Upper Case for the Deployment Flow where it is related to the UI section, otherwise you can use lowercase